src/main/java/com/commercetools/sync/products/utils/ProductVariantUpdateActionUtils.java

```diff
@@ -196,31 +199,27 @@
             final int variantId,
             @Nonnull final List<Image> oldImages,
             @Nonnull final List<Image> newImages) throws IllegalArgumentException {
-
-        if (oldImages.size() != newImages.size()) {
+        final int oldImageListSize = oldImages.size();
+        final int newImageListSize = newImages.size();
```

Comment:
I stored the image sizes in vars since they are used a couple of times down there.